ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -1,1217 +1,27 @@
 presubmits:
   openshift/release:
-  - agent: kubernetes
```

Comment:
@liangxia I found that the code below was deleted when I used the "make" command. Is this normal?

Reply:
Sounds like a rebase issue